src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working.");

```

Comment:
The message in `doWork` method is not fully informative. It should indicate that it is a `Bulldozer` that is working. Consider changing the message to something like 'Bulldozer is working.' to provide more clarity.